.changeset/thick-geckos-reply.md

```diff
@@ -0,0 +1,15 @@
+---
+'@sap-ux/fe-fpm-writer': patch
+'@sap-ux/fiori-elements-writer': patch
+'@sap-ux/fiori-freestyle-writer': patch
+'@sap-ux/logger': patch
+'@sap-ux/odata-service-writer': patch
+'@sap-ux/store': patch
+'@sap-ux/ui5-application-writer': patch
+'@sap-ux/ui5-config': patch
+'@sap-ux/ui5-proxy-middleware': patch
+'test-project': patch
+'@sap-ux/yaml': patch
```

Comment:
That might be a stupid question but why do we need to patch the modules? You only updated `devDependencies` so the resulting built module doesn't change, or do we expect that the `dist` content changes due to the change of the `typescript` version?